### DISCUSSION TITLE
Readarr ?

### DISCUSSION THREAD

**Discussion by lonelyb3ar** [sct/overseerr#2665] [Feature Requests]
Will we be getting Readarr support?

**Comment by TheCatLady** [comment]
No plans for ebook support at this time.

**Reply by snorkrat** [reply]
What about audiobook support? (readarr does audiobooks as well :D)

**Comment by enigmaoftech** [comment]
+1 for readarr integration. It needs dual instance support like sonarr or radarr. 1 for audiobooks and one for regular books.

**Comment by papirov** [comment]
another +1 for readarr support... having overseerr request books for readarr to download and then auto-syncing it into calibre would be magical :)

**Comment by WalterPinkman** [comment]
Recently switched to Overseerr and I'd love to see Readarr integration.

**Comment by oOStroudyOo** [comment]
+1

**Comment by sveatlo** [comment]
+1 readarr support would be amazing

**Comment by gavinmcfall** [comment]
+1 for readarr support should you choose to consider it in the future

**Comment by binarygeek119** [comment]
For metadata there's a new musicbrainz like source being worked on right now called bookbrainz https://bookbrainz.org/

**Comment by xxDmOxx** [comment]
This would be amazing to add to Overseer! I imagine a good number of Plex Server owners are running the Audnexus Plex Audiobook Plugin or AudioBookShelf.
